rtt/transports/corba/CorbaDispatcher.hpp

```diff
@@ -111,7 +113,30 @@ namespace RTT {
                     else
                         name = iface->getOwner()->getName();
                     name += ".CorbaDispatch";
-                    DispatchI[iface] = new CorbaDispatcher( name, scheduler, priority );
+
+                    // The properties to create the CorbaDispatcher are retrieved.
+                    // When the CorbaDispatcher is created these properties can't be changed anymore,
+                    // so they are converted to Constants.
+                    TaskContext* owner = iface->getOwner();
```

Comment:
`owner` might be `NULL`

Reply:
Fixed in https://github.com/orocos-toolchain/rtt/pull/218/commits/285354c5792f8a25c30e4f6b171da531e647ebae.